CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+## v0.17.1
+
+### Changes
+
+- Revert deprecation of the package manifests format. See [#2755](https://github.com/operator-framework/operator-sdk/pull/2755) for deprecation details. The package manifests format is still officially supported by the Operator Framework. ([#3030](https://github.com/operator-framework/operator-sdk/pull/3030), [#3042](https://github.com/operator-framework/operator-sdk/pull/3042), [#3044](https://github.com/operator-framework/operator-sdk/pull/3042))
+
+### Bug Fixes
+
+- Fixes issue where the `helm.operator-sdk/upgrade-force` annotation value for Helm based-operators is not parsed. ([#3027](https://github.com/operator-framework/operator-sdk/pull/3027))
+- In 'run --olm', package manifests format must be replicated in a pod's file system for consistent registry initialization. ([#3038](https://github.com/operator-framework/operator-sdk/pull/3038))
+- the internal OLM client retrieves existing OLM versions correctly now that the returned list of CSVs is indexed properly. ([#3029](https://github.com/operator-framework/operator-sdk/pull/3029))
```

Comment:
```suggestion
- The internal OLM client retrieves existing OLM versions correctly now that the returned list of CSVs is indexed properly. ([#3029](https://github.com/operator-framework/operator-sdk/pull/3029))
```

Reply:
Because this doc is auto-generated, I'm going to leave this as-is then make sure the release notes' formatting is uniform.